### ISSUE TITLE
[BUG] FrostedGlass - Cannot queue multiple uploads

### ISSUE THREAD

**Issue by RikuKawai** [Gibbu/BetterDiscord-Themes#62]
**Describe the bug**
With Frosted Glass enabled, I cannot queue up multiple files for upload. Works as normal with theme disabled, so it's likely not a plugin causing it.

**To Reproduce**
Drag and drop an image or other file on Discord, drag and drop a second image or file.

**Desktop (please complete the following information):**
Discord channel: Stable
OS: Windows 10 Education 1909

**Additional context**
Issue starting occurring after update to Frosted Glass v2.

**Comment by Gibbu** [comment]
I cannot recreate this issue.

**Comment by RikuKawai** [comment]
I just tested it on my laptop (different computer, windows 10 pro 1903) with no plugins loaded and a fresh un-customized install of frosted glass 2.0 and I was able to reproduce the issue exactly as I described it.

The modal about dragging and dropping seems to render under the upload modal and only one file can be queued for upload.

https://www.youtube.com/watch?v=L5_Jy_C9mAA here's a video of me reproducing the bug.
